server/client/src/Routes/PublicRoute.js

```diff
@@ -0,0 +1,11 @@
+import {
+    Navigate
+  } from 'react-router-dom';
+  
+  function PublicRoute({ children, isAuthenticated}) {
```

Comment:
The implementation is not exactly what I wanted, check this [link](https://dev.to/nilanth/how-to-create-public-and-private-routes-using-react-router-72m). This component should return Route or Redirect